services/backend/api_doc/connectivity.js

```diff
@@ -67,6 +67,23 @@
  * }
  */
 
+/**
+ * @apiDefine TimelineByCategoryExample
+ * @apiSuccessExample {json} Success-Example:
+ * [
```

Comment:
Why the result is an array and not an object? If it's only returning the information for a category it should be an object

Reply:
it was changed now is an object.